CHANGELOG.md

```diff
@@ -1,24 +1,33 @@
-# Changelog
-
-* In general follow (https://docs.npmjs.com/getting-started/semantic-versioning) versioning.
-
-## 2.0.3
-* Update react and react-dom packages
-
-## 2.0.2
-* Update @opuscapita/react-icons
-
-## 2.0.1
-* Update react-intl and redux packages
-
-## 2.0.0
-* Upgrade to react 15.6 & babel-preset-env
-
-## 1.1.1
-* Change alertSuccess timeout to 1.5s
-
-## 1.1.0
-* Add AlertsProvider
-
-## 1.0.0
-* Extracted to own repository from oc-common-ui
+# Changelog
+
+* In general follow (https://docs.npmjs.com/getting-started/semantic-versioning) versioning.
+
+## <next>
+* Upgrade to new react-component-layout
```

Comment:
probably should be react-component-template (to not confuse this with common-layouts)